numpy/core/arrayprint.py

```diff
@@ -114,6 +114,7 @@ def set_printoptions(precision=None, threshold=None, edgeitems=None,
     threshold : int, optional
         Total number of array elements which trigger summarization
         rather than full repr (default 1000).
+	Preferred max value to be used is sys.maxsize.
```

Comment:
This phrasing isn't the best. Perhaps ``To always use the full repr without summarization, pass `sys.maxsize`.``

Reply:
Done. Thank you for the suggestion!